### PR TITLE
rox bugfix for ptheme re #2277

### PR DESCRIPTION
**Pull request by 01micko**
if rox is running and theme is changed rox is restarted.

### CODE REVIEW

**Comment by dimkr** [comment]
How did it work in old Puppies in the 5.x and 6.x era? 🤔

**Comment by 01micko** [comment]
> How did it work in old Puppies in the 5.x and 6.x era?

It didn't.

File: woof-code/rootfs-packages/ptheme/usr/sbin/diricons.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ln_diricon() {
+	EPATH=$1
+	ICON=$2
+	[ -e "$ICON" ] && \
+		(cd "$EPATH"; ln -svf "$ICON" ./.DirIcon )
+}
+
+for icon in $(ls "/usr/share/icons/$ICON_THEME/48/places/" | grep "^directory");do
+	if echo $icon|grep -q 'directory-home' ;then
+			ln_diricon "$HOME" "/usr/share/icons/$ICON_THEME/48/places/$icon"
```
Comment (dimkr) [inline woof-code/rootfs-packages/ptheme/usr/sbin/diricons.sh]: Is it possible to link to the .svg and let ROX-Filer resize to 48 when it renders the icon?

Comment (01micko) [reply woof-code/rootfs-packages/ptheme/usr/sbin/diricons.sh]: All icons _are_ svg in the native puppy themes.

**Pull request merged**